docs/behat/basic-usage.rst

```diff
@@ -0,0 +1,544 @@
+Basic Usage
+===========
+
+The best way of understanding how things work in detail is showing and analyzing examples, that is why this section gathers all the knowledge from the previous chapters.
+Let's assume that we are going to implement the functionality of managing countries in our system.
+Now let us show you the flow.
+
+Describing features
+-------------------
+First of all let's start with writing our feature file the most crucial thing here is ubiquitous language. We don't want to specify how something is done,it doesn't really matter in case of business.
```

Comment:
I am not sure what you mean here. Ubiquitous Language is about building a common language between users and developers. http://martinfowler.com/bliki/UbiquitousLanguage.html

Perhaps something like:

```
Let's start with writing our feature file, which will contain answers to the most important questions: Why (benefit), who (actor using the feature) and what (the feature itself). It should also include scenarios, which serve as examples of how things supposed to work.
```